client/src/components/JsonSchemaBuilder/components/SchemaInput.tsx

```diff
@@ -0,0 +1,58 @@
+import {Input} from '@/components/ui/input';
+import {Label} from '@/components/ui/label';
+import React, {FormEvent, useEffect, useState} from 'react';
+import {useDebouncedCallback} from 'use-debounce';
+
+const handleStringChange = (handler: (value: string) => void) => {
+    return (event: FormEvent<HTMLElement>): void => {
+        handler((event.target as HTMLInputElement).value);
+    };
+};
```

Comment:
ne moraš to sad minjat, ali arrow funkcija ima implicitni return (`=>` je skraćenica za return) tako da kad imaš kod kao u ovoj funkciji, možeš slobodno izbacit vitičaste zagrade i return